DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
+    & Conditional Pass & Missing git-lfs in step 2. Used docker image instead. Remaining steps
+pass.
+
+    \\\hline
+    \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T218}{LVV-T218} 
+    & Conditional Pass & Missing git-lfs in step 2 and 3. Used docker image instead. Last step
+result deviated sligthly from the expected result for one consumer.
 
-\begin{longtable} {|p{0.2\textwidth}|p{0.2\textwidth}|p{0.6\textwidth}|}\hline
-{\bf TEST CASE ID} & {\bf PASS/FAIL} & {\bf COMMENTS} \\\hline
-LVV-T216 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T217 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Remaining steps pass.} \\\hline
-LVV-T218 & Conditional Pass & {\textit Missing git-lfs in step 2 and 3. Used docker image instead. Last step result deviated slightly from the expected result for one consumer.} \\\hline
+    \\\hline
+
+\caption{Test Results Summary}
+\label{table:summary}
 \end{longtable}
 
 \subsection{Overall Assessment}
 \label{sect:overallassessment}
 
-The test campaign is considered in overall: PASS.
-
-Missing elements on the Kubernetes Commons prevented us from building the Docker images in place and using a complete night of unique alerts.
-However, using pre-built Docker images with a smaller, repeated sample of alerts enabled the test to proceed as expected.
+The test campaign is considered in overall:
+PASS.\\[2\baselineskip]Missing elements on the Kubernetes Commons
+prevented us from building the Docker images in place and using a
+complete night of unique alerts. However, using pre-built Docker images
+with a smaller, repeated sample of alerts enabled the test to proceed as
+expected.
 
 
 \subsection{Recommended Improvements}
 \label{sect:recommendations}
 
-Re-execution with the missing components on the Kubernetes Commons.
+Re-execution with the missing components on the Kubernetes
+Commons.\\[2\baselineskip]Examination of the deserialized alerts in the
+logs is cumbersome and does not allow real verification of the alert
+content. Improved tools for counting alerts, comparing their contents to
+the sent alerts, and timing throughput would all improve the utility of
+this test.
 
-Examination of the deserialized alerts in the logs is cumbersome and does not allow real verification of the alert content. 
-Improved tools for counting alerts, comparing their contents to the sent alerts, and timing throughput would all improve the utility of this test.
 
+\newpage
+\section{Detailed Test Results}
+\label{sect:detailedtestresults}
 
 
+  \subsection{Test Cycle LVV-C3 }
 
-\newpage
+Open test cycle {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testrun/LVV-C3}{LDM-503-5: Alert Distribution Validation}} in Jira.
 
-\section{Detailed Test Results}
-\label{sect:detailed}
+  LDM-503-5: Alert Distribution Validation\\
+  Status: Done
+
+  Test run with test cases in draft.
```

Comment:
See my previous comment re “draft”.

Reply:
This text is the Description of LVV-C3, that should be probably improved.

Having test case in "draft" is a procedural problem (see confluence page https://confluence.lsstcorp.org/display/DM/DM+Test+Approach)